src/services/user-service.js

```diff
@@ -230,8 +230,8 @@ async function _generateAccessToken(transaction) {
 }
 
 function _verifyEmailActivation(emailActivated) {
-  const emailActivation = Config.get("Email:ActivationEnabled") || 'off';
-  if (emailActivation === 'on' && emailActivated === 0)
+  const emailActivation = Config.get("Email:ActivationEnabled");
+  if (emailActivation && emailActivated === 0)
```

Comment:
why do you need emailActivated === 0? config setting is of type boolean

Reply:
change emailActivated to boolean

Reply:
fixed